src/main/java/seedu/address/model/ReadOnlyUserPrefs.java

```diff
@@ -27,6 +27,12 @@
      */
     Alias getAlias(String aliasName);
 
+    /**
+     * Deletes an user-defined alias from the current mapping.
+     * @param aliasName The name of the alias to be deleted.
```

Comment:
Should there be a blank line between description and parameters?

Reply:
Done!